### ISSUE TITLE
Wunjo AI Face and Mouth Animation Results in Cannot Find File Specified Error

### ISSUE THREAD

**Issue by RaisingAwesome** [wladradchenko/wunjo.wladradchenko.ru#35]
Running the Face and Mouth Animation results in "The system cannot find the file specified" error.  All else looks like it ran fine until it hits a warning about the image size.

**Log:**
Error ... [WinError 2] The system cannot find the file specified
WARNING:imageio_ffmpeg:IMAGEIO FFMPEG_WRITER WARNING: input image is not divisible by macro_block_size=16, resizing from (256, 255) to (256, 256) to ensure video compatibility with most codecs and players. To prevent resizing, make your input image divisible by the macro_block_size or set the macro_block_size to 1 (risking incompatibility).
Face Renderer:: 100%|##########| 80/80 [06:28<00:00, 4.86s/it]
Face Renderer:: 100%|##########| 80/80 [06:28<00:00, 4.84s/it]
...

**Comment by wladradchenko** [comment]
Hello. It looks like you are using either a very small image or an unsupported format.  I don’t think it’s a bug or necessary to support all formats and sizes.  Change resolution minimum 512 and maximum 1920 and use formats mp4, png, jpg, jpeg.

**Comment by RaisingAwesome** [comment]
I resized to 1980 and ensured it is a jpg.  It removed the warning, but still states "Error ... [WinError 2] The system cannot find the file specified".  An MP4 file is created, but gives a message that it is only 18KB and video players state it has a problem and cannot play it.

This is on a Windows 11 machine.  When the error is thrown, two windows open up rapidly and close immediately.  It appears the error happens with whatever is happening at the point of all frames being completed as shown below.

Error ... [WinError 2] The system cannot find the file specified
Face Renderer:: 100%|##########| 32/32 [02:51<00:00, 5.37s/it]
Face Renderer:: 100%|##########| 32/32 [02:51<00:00, 4.94s/it]
Face Renderer:: 97%|#########6| 31/32 [02:46<00:04, 4.93s/it]
Face Renderer:: 94%|#########3| 30/32 [02:41<00:09, 4.89s/it]
Face Renderer:: 91%|######### | 29/32 [02:36<00:14, 4.84s/it]
Face Renderer:: 88%|########7 | 28/32 [02:31<00:19, 4.78s/it]
Face Renderer:: 84%|########4 | 27/32 [02:27<00:23, 4.79s/it]
Face Renderer:: 81%|########1 | 26/32 [02:22<00:29, 4.87s/it]
Face Renderer:: 78%|#######8 | 25/32 [02:17<00:34, 4.93s/it]
Face Renderer:: 75%|#######5 | 24/32 [02:13<00:40, 5.03s/it]
Face Renderer:: 72%|#######1 | 23/32 [02:08<00:46, 5.16s/it]
Face Renderer:: 69%|######8 | 22/32 [02:03<00:52, 5.27s/it]
Face Renderer:: 66%|######5 | 21/32 [01:58<00:59, 5.41s/it]
Face Renderer:: 62%|######2 | 20/32 [01:53<01:08, 5.67s/it]
Face Renderer:: 59%|#####9 | 19/32 [01:47<01:13, 5.62s/it]
Face Renderer:: 56%|#####6 | 18/32 [01:42<01:19, 5.64s/it]
Face Renderer:: 53%|#####3 | 17/32 [01:36<01:24, 5.62s/it]
Face Renderer:: 50%|##### | 16/32 [01:31<01:30, 5.65s/it]
Face Renderer:: 47%|####6 | 15/32 [01:25<01:36, 5.66s/it]
Face Renderer:: 44%|####3 | 14/32 [01:19<01:42, 5.68s/it]
Face Renderer:: 41%|#### | 13/32 [01:14<01:47, 5.65s/it]
Face Renderer:: 38%|###7 | 12/32 [01:08<01:53, 5.66s/it]
Face Renderer:: 34%|###4 | 11/32 [01:02<01:59, 5.67s/it]
Face Renderer:: 31%|###1 | 10/32 [00:57<02:04, 5.65s/it]
Face Renderer:: 28%|##8 | 9/32 [00:51<02:12, 5.76s/it]
Face Renderer:: 25%|##5 | 8/32 [00:45<02:17, 5.71s/it]

**Comment by wladradchenko** [comment]
Have you tried to animate a face using a picture from [here](https://github.com/wladradchenko/wunjo.wladradchenko.ru/tree/main/example/test)?  Do you have the internet icon green enabled?  When opening a module, is there a message saying that some model was not found?  What is browser you use: your default or my WebGUI? What is version of app you use? 

About the sudden opening of the window, which you say is a launch ffmpeg in terminal.

**Comment by RaisingAwesome** [comment]
Thanks for the suggestions.

I tried the suggestions with the example image and got the same error.
I adjusted permissions on the folder.

I run it by "C:\Users\Sean-\OneDrive\Apps\Program Files\Wunjo\wunjo\src\Wunjo Wladradchenko AI.exe" which appears to work with all other features.

Is there a way to run it with Python versus the .exe?

Thanks for your time on this.  It is much appreciated.

-Sean

**Comment by wladradchenko** [comment]
You can read wiki [page](https://github.com/wladradchenko/wunjo.wladradchenko.ru/wiki/How-to-install-the-application) about using the code by python.

**Comment by RaisingAwesome** [comment]
Thanks for all your help.  After trying all kinds of things, I found the first clue held the answer - the two screens that popped up that you said were for ffmpeg.  

I had a prior installation of ffmpeg installed and referenced in my PATH variable.  I believe when it was attempting to run it, it was not successful.  So, I eliminated the reference in my PATH variable so that Wunjo would install it's own ffmpeg.  This solved it.

Thanks so much!  Amazing work you have done!

-Sean
